views/course_view.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.actions.act_window" id="open_academy_course_action">
+        <field name="name">Courses</field>
+        <field name="res_model">course</field>
+        <field name="view_mode">tree,kanban,form</field>
+    </record>
+
+    <record model="ir.ui.view" id="open_academy_course_view_action">
+        <field name="name">List courses</field>
+        <field name="model">course</field>
```

Comment:
You need to try your changes, this should cause an error when updating the module. Your model name is "open.academy.course"